src/App.css

```diff
@@ -39,10 +48,15 @@
   padding: 5px 0 20px 0;
 }
 
-tr {
+tr,
+.not-selected {
   color: #5b5757;
 }
 
+.selected {
+  background-color: aqua;
+}
```

Comment:
Good use of CSS classes to add a highlight to a row, to make this even better, can you work out how to add an onHover so that you can still distinguish which rows are highlighted, even while hovering over them?